e2e/test/iothub/service/RegistryManagerE2ETests.cs

```diff
@@ -87,6 +88,62 @@ public async Task RegistryManager_AddDeviceWithTwinWithDeviceCapabilities()
             }
         }
 
+        [LoggedTestMethod]
+        public async Task RegistryManager_BulkLifecycle()
+        {
+            int bulkCount = 50;
+            string[] deviceIds = new string[bulkCount];
+            List<Device> devices = new List<Device>();
+            for (int i = 0; i < bulkCount; i++)
+            {
+                deviceIds[i] = _devicePrefix + Guid.NewGuid();
+                devices.Add(new Device(deviceIds[i]));
+            }
+
+            var registryManager = RegistryManager.CreateFromConnectionString(Configuration.IoTHub.ConnectionString);
+
+            // Test that you can create devices in bulk
+            var bulkAddResult = await registryManager.AddDevices2Async(devices).ConfigureAwait(false);
+            Assert.IsTrue(bulkAddResult.IsSuccessful);
+
+            foreach (Device device in devices)
+            {
+                // After a bulk add, every device should be able to be retrieved
+                Assert.IsNotNull(await registryManager.GetDeviceAsync(device.Id).ConfigureAwait(false));
```

Comment:
This is totally fine but if we are following fluent assertions in general, we should be consistent and use it. Ex: .Should().Be()

Reply:
We don't really use that pattern in this repo for whatever reason. I'd rather keep the pattern here